src/projects/migrations/0001_initial.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.1.5 on 2021-02-03 13:44
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('categories', '0004_auto_20200217_1136'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Project',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(db_index=True, max_length=256, verbose_name='Name')),
+                ('slug', models.SlugField(blank=True, help_text='Let it empty so it will be autopopulated.', verbose_name='Slug')),
+                ('description', models.TextField(blank=True, default='', verbose_name='Full description of the backer')),
```

Comment:
```suggestion
                ('description', models.TextField(blank=True, default='', verbose_name='Full description of the project')),
```

Et du coup rajouter une entrée dans le django.po + modifier le models.py